lib/BackgroundJobs/IndexerJob.php

```diff
@@ -125,7 +125,7 @@ protected function index(array $files): void {
 			$userIds = $this->storageService->getUsersForFileId($queueFile->getFileId());
 			foreach ($userIds as $userId) {
 				try {
-					$source = new Source($userId, 'file: ' . $file->getId(), $file->getPath(), $fileHandle, $file->getMtime(), $file->getMimeType());
+					$source = new Source($userId, 'file: ' . $file->getId(), $file->getPath(), $fileHandle, $file->getMtime(), $file->getMimeType(), 'files');
```

Comment:
shouldn't `file:` and `files` be the same then, as well?

Reply:
yeah that would make it consistent :+1: